applications/RomApplication/python_scripts/calculate_rom_basis_output_process.py

```diff
@@ -58,10 +58,10 @@ def __init__(self, model, settings):
 
         # Set the ROM basis output settings
         self.rom_basis_output_format = settings["rom_basis_output_format"].GetString()
-        if not self.rom_basis_output_format == "json":
-            err_msg = "Provided \'rom_basis_output_format\' is {}. Available options are \'json\'.".format(self.rom_basis_output_format)
+        rom_basis_output_available_formats = ["json", "numpy"]
+        if self.rom_basis_output_format not in rom_basis_output_available_formats:
+            err_msg = "Provided \'rom_basis_output_format\' is {}. Available options are \'json\' and \'numpy\'.".format(self.rom_basis_output_format)
             raise Exception(err_msg)
-        self.rom_basis_output_name = settings["rom_basis_output_name"].GetString()
```

Comment:
In my opinion, it would be beneficial to retain the choice of "rom_basis_output_name". For example, in the context of CoSimulation for fluid-thermal coupling, it is necessary to have separate files named RomParameters_fluid.json and RomParameters_thermal.json. Therefore, I propose allowing the ability to pass the name of the "RomParameters" file to the ``RomAnalysis`` (this can be done in a future PR).